.github/workflows/lint.yml

```
@@ -0,0 +1,20 @@
+name: Lint workflow
+
+on: [pull_request]
+
+jobs:
+  lint-black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Есть более свежая версия
```suggestion
      - uses: actions/checkout@v3
```

Давай сформируем общее правило оформления для шагов такое: каждый шаг должен содержать name. Так мы сможем более гибко управлять своими шагами (сможем задавать наши имена, а не стандартные). Тогда каждый шаг нужно переписать в таком ключе
```suggestion
-   name: Checkout code
    uses: actions/checkout@v3
```

---

Спасибо за подробные комментарии, с правками согласен, изменения внес :)